ui: Exclude Service Health from Node listing page

johncowen opened the pull request:
This PR excludes the health of any service instances from the Node Listing page. This means that if you are viewing the Node listing page you will only see failing nodes if there are any Node Checks failing, Service Instance Health checks are no longer taken into account.

jgwhite commented on ui/packages/consul-ui/app/models/node.js:
```diff
@@ -45,16 +47,16 @@ export default class Node extends Model {
 
   @computed('Checks.[]')
```
Worth updating these dependency lists to `NodeChecks.[]`?

Also: I’m surprised these are `@computed` decorators are necessary, but I shall trust that they are.

jgwhite commented on ui/packages/consul-ui/tests/acceptance/dc/nodes/index.feature:
```diff
@@ -7,6 +7,36 @@ Feature: dc / nodes / index
     body: |
       "211.245.86.75:8500"
     ---
+  Scenario: Viewing an a node with an unhealthy NodeCheck
```
```suggestion
  Scenario: Viewing a node with an unhealthy NodeCheck
```

jgwhite commented on ui/packages/consul-ui/tests/acceptance/dc/nodes/index.feature:
```diff
@@ -7,6 +7,36 @@ Feature: dc / nodes / index
+    Then the url should be /dc-1/nodes
+    Then I see 1 node models
+    And I see status on the nodes.0 like "critical"
+  Scenario: Viewing an a node with an unhealthy ServiceCheck
```
```suggestion
  Scenario: Viewing a node with an unhealthy ServiceCheck
```

jgwhite reviewed (approved):
Looks good to me. One question and a couple of nitpicks, but all good apart from that 👍

johncowen commented:
K @jgwhite I fixed those deps up and added a Changelog, I'll leave this hanging until tomorrow just incase you had anything else and then go ahead and merge. Thanks for the review! 🙌

johncowen commented:
gettin' on the merge train 🚋 🚋

hc-github-team-consul-core commented:
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/580568.
